<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWndShow</class>
 <widget class="QWidget" name="QWndShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QWndShow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>主机功率</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ledPower"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>海水温度</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLCDNumber" name="lcdTemp"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>淡水温度</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLCDNumber" name="lcdInTemp">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>低温淡水进中冷器温度</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLCDNumber" name="lcdOutTemp">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>低温淡水出中冷器温度</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tabData">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <row>
      <property name="text">
       <string>额定</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>No.1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>No.2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>No.3</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>流量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>频率</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>转速</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>流量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>扬程</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
